.gitignore

```diff
@@ -2,7 +2,7 @@ venv
 es-venv
 
 # IDE junk
-.idea
+.idea/dictionaries
```

Comment:
The value `.idea` here was correct: it tells git to ignore all the files and folders inside `.idea` folder.
If we change it to `.idea/dictionaries`, then only `dictionaries` will be ignored, and all the other contents of `.idea` folder will be tracked by git. Please, change it back to `.idea`.

`.idea/dictionaries` directory was added to the repository before `.gitignore` was created. So, git did not remove the dictionaries, even thow they were supposed to be ignored.